packages/enterprise/src/license.ts

```diff
@@ -580,21 +580,25 @@ export async function postResendEmailVerificationEmailToLicenseServer(
   );
 }
 
-// Creates or updates the license in the MongoDB cache in case the license server goes down.
-async function createOrUpdateLicenseMongoCache(license: LicenseInterface) {
-  await LicenseModel.findOneAndUpdate(
-    { id: license.id },
-    { $set: license },
-    { upsert: true }
-  );
+// Creates or replaces the license in the MongoDB cache in case the license server goes down.
+async function createOrReplaceLicenseMongoCache(
+  license: LicenseInterface | LicenseDocument
```

Comment:
A `LicenseDocument` object should never make it this far.  I find it best to never use Mongoose objects.  If you get one back from a `find` operation, just immediately convert it to raw JSON.  We have helper methods like this in the app for all of our data models:

```ts
function toInterface(doc: FactTableDocument): FactTableInterface {
  const ret = doc.toJSON<FactTableDocument>();
  return omit(ret, ["__v", "_id"]);
}
```